test/boulder-tools/tag_and_upload.sh

```diff
@@ -19,7 +19,7 @@ do
   # would be cleaner if we could just output the `sed` directly to the `docker
   # build` stdin but that requires Docker 17+ too! :'(
   DOCKERFILE="golang.$GO_VERSION.Dockerfile"
-  sed -r \
+  gsed -r \
```

Comment:
Did you mean to include this diff? It looks specific to a MacOS env if I had to guess.

Reply:
D'oh, yup, keep meaning to just alias sed -> gsed but I never get around to it.